contrib/motd

```diff
@@ -1,32 +1,32 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
 import socket
-import executil
+from subprocess import run, PIPE
+from sys import stdin
+
 from sysversion import fmt_sysversion
 
-def fmt_welcome():
-    welcome = "Welcome to %s" % socket.gethostname().capitalize()
+encoding = stdin.encoding # most likely 'utf-8'
```

Comment:
might make for clearer code if this were `ENCODING` rather than `encoding` to make it clear it's a global

Reply:
Actually, that file is still python2! Oops...